docs/api/desktop-capturer.md

```diff
@@ -1,7 +1,7 @@
 # desktopCapturer
 
-The `desktopCapturer` module can be used to get available sources that can be
-used to be captured with `getUserMedia`.
+> Capture audio, video, and images from a microphone, camera, or
```

Comment:
`desktopCapturer` lists sources to capture for `getUserMedia`, instead of doing the capture itself.